### PR TITLE
Add PileMd.app v0.9

### PR DESCRIPTION
**Pull request by exadeci**

### CODE REVIEW

File: Casks/pilemd.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'pilemd' do
+  version '0.9.0'
+  sha256 '979437c71a54372f29c87f64ae4bc784f358fa10a810f6c5c786885f2e4a087a'
+
+  url "https://github.com/hirokiky/pilemd/archive/v#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/pilemd.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/pilemd.rb]: Needs a `# github.com/hirokiky/pilemd was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/pilemd.rb]: This will download the source code, not the app. Use the link on the homepage.

**Comment by exadeci** [comment]
Fixed the url and added the appcast

File: Casks/pilemd.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'pilemd' do
+  version '0.9.0'
+  sha256 '979437c71a54372f29c87f64ae4bc784f358fa10a810f6c5c786885f2e4a087a'
+
+  url "https://s3-ap-northeast-1.amazonaws.com/pile-md-dist/pilemd.darwin-x64.#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/pilemd.rb]: Needs a `# s3-ap-northeast-1.amazonaws.com/pile-md-dist was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8ae554fa8284e0309a625da0d45208064e13134f. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**